aspnetcore/blazor/webassembly-lazy-load-assemblies.md

```diff
@@ -41,6 +41,14 @@ Only assemblies that are used by the app can be lazily loaded. The linker strips
 > </ItemGroup>
 > ```
 
+## Prerendering
```

Comment:
I feel like this might work better as a call-out note under the "Assembly load logic in `OnNavigateAsync`" section below so that it is in context with the rest of material on the service.